plugins/main/public/controllers/management/components/management/cluster/cluster-overview.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import { compose } from 'redux';
+import {
+  withErrorBoundary,
+  withGlobalBreadcrumb,
+  withReduxProvider,
+  withUserAuthorizationPrompt,
+} from '../../../../../components/common/hocs';
+import { cluster } from '../../../../../utils/applications';
+import { AppState, WzRequest } from '../../../../../react-services';
+import { ClusterDisabled } from '../../../../../components/management/cluster/cluster-disabled';
+import { ClusterDashboard } from '../../../../../components/management/cluster/dashboard/dashboard';
+
+interface ClusterOverviewState {
+  authorized: boolean;
+  clusterEnabled: boolean;
+  isClusterRunning: boolean;
+  statusRunning: string;
+  permissions: any;
+}
+
+export const ClusterOverview = compose(
+  withErrorBoundary,
+  withReduxProvider,
+  withGlobalBreadcrumb([{ text: cluster.breadcrumbLabel }]),
+  withUserAuthorizationPrompt([
+    { action: 'cluster:status', resource: '*:*:*' },
+  ]),
+)(
+  class ClusterOverview extends Component<unknown, ClusterOverviewState> {
+    _isMount = false;
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        authorized: true,
+        clusterEnabled: false,
+        isClusterRunning: false,
+        statusRunning: 'no',
+        permissions: undefined,
+      };
+    }
+
+    clusterStatus = async () => {
+      try {
+        const status = await WzRequest.apiReq('GET', '/cluster/status', {});
+        this.setState({
+          authorized: true,
+        });
+        return status;
+      } catch (error) {
+        if (error === '3013 - Permission denied: Resource type: *:*')
+          this.setState({
+            authorized: false,
+          });
```

Comment:
**thought**: this could not be necessary due to the permission check that wraps this component with `withUserAuthorizationPrompt`.

Reply:
The `clusterStatus` method is also removed and the `WzRequest` is left directly in `componentDidMount` to obtain the status.